api/handlers_mobile.go

```diff
@@ -784,3 +793,91 @@ func MobileSearchHandler(c *gin.Context) {
 		NewInternalError(err).Abort(c)
 	}
 }
+
+// call feed api to get results like search - https://kabbalahmedia.info/feed_api/feed
```

Comment:
comment is not clear, remove or change to something like:
Invoke Feed API and natives results for mobile app